src/subtyping.md

```diff
@@ -15,5 +15,70 @@ fn bar<'a>() {
     let t: &'a str = s;
 }
 ```
-Since `'static` "lives longer" than `'a`, `&'static str` is a subtype of
-`&'a str`.
+
+Since `'static` outlives `'a`, `&'static str` is a subtype of `&'a str`.
```

Comment:
If it's not declared above, could we clarify that `'a` here is a standin for "any generic lifetime in scope" (or maybe just "some lifetime"?)